django/apps/blue_management/blue_mgnt/views/policies.py

```diff
@@ -210,13 +210,8 @@ def _add_inherit_from_field(self):
         """ Create a TypedChoiceField that coerces to integer and uses
         api.list_policies to provide inheritance choices """
 
-        self.fields['inherit_from'] = forms.TypedChoiceField(
-            required=False,
-            choices=self._parent_choices(),
-            empty_value=None,
-            coerce=int)
-
-        self.fields['inherit_from'].initial = self._inherit
+        self.fields['inherit_from'] = forms.IntegerField(
+            required=False, widget=forms.HiddenInput, initial=self._inherit)
```

Comment:
Yes. When we copy from an existing policy this is how the inheritance is specified.